chapters/introduction/lab/content/explorer/transactions.md

```diff
@@ -8,7 +8,7 @@ We can see that there are 6 transactions in this group.
 
 ## Miniblocks
 
-Transactions are grouped in **miniblocks**.
+Cross-shard transactions are grouped in **miniblocks**.
```

Comment:
Maybe also give some references here to the official documentation, an example or provide more information about this?

Reply:
About cross-shard transactions or mini-blocks?

Reply:
Both, how they relate to each other. One line or the example in the documentation should be enugh.